packages/core/src/Modules/Cashier/Components/withdraw.jsx

```diff
@@ -3,22 +3,15 @@ import React from 'react';
 import { connect } from 'Stores/connect';
 import CashierContainer from './cashier-container.jsx';
 
-class Withdraw extends React.Component {
-    componentDidMount() {
-        this.props.setActiveTab(this.props.container);
-        this.props.onMount(this.props.verification_code);
-    }
+const Withdraw = ({ container, iframe_height, iframe_url, is_loading, onMount, setActiveTab, verification_code }) => {
+    React.useEffect(() => {
+        setActiveTab(container);
+        onMount(verification_code);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Can't we just add the `deps` instead of using doing `eslint-disable-next-line`?

Reply:
We just want a `didMount` behavior so I think better to not include them as deps.

Reply:
Why not?

Reply:
In the parent component we've added these as `deps`. Let's do the same here. :+1: